packages/packages/libs/editor-controls/src/components/css-code-editor/css-editor.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { styled } from '@elementor/ui';
+
+export const EditorWrapper = styled( 'div' )`
```

Comment:
can we use React elements here?
```suggestion
export const EditorWrapper = styled( Box )`
```